proto/storage/proto.lock

```diff
@@ -4228,7 +4307,7 @@
                 "options": [
                   {
                     "name": "(gogoproto.moretags)",
-                    "value": "search:\"Compliance Profile Name,hidden,store\""
+                    "value": "search:\"Compliance Profile Name,hidden\""
```

Comment:
does it have an impact to remove the store search annotation ?

Reply:
Lock just represents what is defined in proto files. I'm not able to answer your question, because I'm not sure what this annotation represents.

It was changed in this PR: https://github.com/stackrox/stackrox/pull/9013

Reply:
@mtodor `store` is no longer necessary or even used.  At some point when we get a minute it will disappear from all protos.